packages/git/src/common/git-model.ts

```diff
@@ -191,6 +191,27 @@ export namespace Repository {
     export const sortComparator = (ra: Repository, rb: Repository) => rb.localUri.length - ra.localUri.length;
 }
 
+/**
+ * Representation of a Git remote.
+ */
+export interface Remote {
```

Comment:
```ts
interface Remote {
  name: string;
  fetch: string;
  push: string;
}
```

Thoughts?

Reply:
Yes ! I think that's much nicer thank you!